docs/DeveloperGuide.md

```diff
@@ -190,9 +190,75 @@ call to update the value of the corresponding field of `contact`.
 ![](images/ContactSetFieldsSequenceSubdiagram.png)
 
 ### Load and Store
-Coming soon.
 
+**API:** `StorageFile.java`
 
+![](images/StorageFileClassDiagram.png)
+
+Note: Exception classes are left out of this diagram that aims to show the core structure of the load-save functionality.
+Some methods and attributes are not mentioned to keep the diagram simple while keeping the core information visible.
+
+As seen from the class diagram above (which shows the portions relevant to the StorageFile class), 
+the StorageFile class associates to a Logger class, a ContactsManager class, an ExpenseTracker class
+and a StudyManager class. The AllOnUs class (to do with main menu) associates to a StorageFile class and the 
+manager and tracker classes associate to the StorageFile class as well.
+
+The StorageFile class has private attributes `fileName` and `dataFileRelativePath` which define the name of the file
+and the relative path of the file to project directory, which are utilized in creating/locating the file by the 
+StorageFile class. More details on this will be discussed soon. All the methods in this class are public and the
+only static method `setFields()` is utilized by class AllOnUs to initialize the StorageFile class with instances of
+the ContactsManager, ExpenseTracker and StudyManager classes used in the application and the file related attributes.
+
+`get*InFileFormat()`, `load*()` and `create*()` methods represent multiple methods that adhere to the respective forms
+and will be discussed further as needed. 
```

Comment:
Consider also editing the main menu section so that the formatting is consistent with this section!